code/game/objects/items/weapons/tanks/jetpack.dm

```diff
@@ -27,7 +27,6 @@
 	. = ..()
 	if(air_contents.total_moles < 5)
 		user << "<span class='danger'>The meter on \the [src] indicates you are almost out of gas!</span>"
-		playsound(user, 'sound/effects/alert.ogg', 50, 1)

```

Comment:
This does solve it. It plays the sound and shows the warning upon someone examining it. And since ghosts and AI's can examine it, they can spam the examine verb on the item and play the sound repeatedly. The sound does not play automatically upon going to low gas.